CMakeLists.txt

```diff
@@ -118,9 +118,9 @@ if( EXIV2_BUILD_SAMPLES )
             WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
             COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose tiff_test
         )
-        add_test(NAME lensTests
+        add_test(NAME versionTest
             WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests
-            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose lens_tests
+            COMMAND cmake -E env EXIV2_BINDIR=${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${Python3_EXECUTABLE} runner.py --verbose bash_tests/version_test.py 
```

Comment:
Why is there a need for this, isn't version_test.py run as part of bashTests above already? I see below you wanted to have just the minimal versionTest w/o samples/unit tests (good idea), but then it should be addition, not removal of lensTests, no?

Reply:
That's a mistake.  I'll restore lensTest tomorrow.

I find the version test very useful.   When I think "what have we got here", my finger rattle out `$ make version_test`.  In future it'll be `$ ctest -R version`.  In DOS, I have to type`$ ctest -R version -C Release`.  The people at Kitware didn't think out the `ctest -C option` very well.